pyiron/atomistics/job/atomistic.py

```diff
@@ -508,14 +508,18 @@ def get_structure(self, iteration_step=-1, wrap_atoms=True):
         if not (self.structure is not None):
             raise AssertionError()
         snapshot = self.structure.copy()
-        snapshot.cell = self.get("output/generic/cells")[iteration_step]
-        snapshot.positions = self.get("output/generic/positions")[iteration_step]
-        indices = self.get("output/generic/indices")
-        if indices is not None:
+        snapshot.cell = self.output.cells[iteration_step]
+        snapshot.positions = self.output.positions[iteration_step]
+        indices = self.output.indices
+        if indices is not None and len(indices)>max([iteration_step,0]):
             snapshot.indices = indices[iteration_step]
         if wrap_atoms:
             return snapshot.center_coordinates_in_unit_cell()
         else:
+            if len(self.output.unwrapped_positions)>max([iteration_step,0]):
+                snapshot.positions = self.output.unwrapped_positions[iteration_step]
+            else:
+                snapshot.positions += self.output.total_displacements[iteration_step]
```

Comment:
Is `self.output.total_displacements` implemented for all types of jobs? (LAMMPS, VASP and SPHINX)

Reply:
This comes from `GenericOutput` in `atomistic` so it should be there by default